### ISSUE TITLE
Could you share your training parameters?

### ISSUE THREAD

**Issue by huangbiubiu** [pppoe/tensorflow-sphereface-asoftmax#6]
I tried to reproduce sphereface via tensorflow on CASIA-WebFace, but I found the loss function is very difficult to convey, and loss function sometimes be very large (10e6). Are there any solutions?
